upa/src/sdk/pointCompression.ts

```diff
@@ -51,6 +54,10 @@ export function decompressG1Point(compressedPoint: CompressedG1Point): G1Point {
   const x3: bigint = FQ.mul(x2, x);
   const x3PlusB: bigint = FQ.add(x3, BN254_G1_B);
   const y: bigint = FQ.sqrt(x3PlusB);
+  if (!y) {
+    return undefined;
+  }
+
   assert("bigint" == typeof y, "sqrt failed");
```

Comment:
Probably can take this assert out (to take out the crashing path)- I think y could only be have been null or bigint.

Reply:
done